tools/src/bin/rust-indexer.rs

```diff
@@ -331,7 +333,7 @@ fn analyze_file(
         let def = match defs.get(file_analysis, ref_.ref_id) {
             Some(d) => d,
             None => {
-                eprintln!("Dropping ref {:?}, kind {:?}, no def", ref_.ref_id, ref_.kind);
+                debug!("Dropping ref {:?}, kind {:?}, no def", ref_.ref_id, ref_.kind);
```

Comment:
I'd prefer if a couple of these would be `warn!` ings instead, but if they clobber the log a lot `debug!` is fine.